xnmt/inference.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, model_file=None, src_file=None, trg_file=None, ref_file=None,
     self.len_norm_type = len_norm_type
     self.mode = mode
     self.batcher = batcher
-    
+
```

Comment:
I actually removed the unnecessary whitespace characters by executing the script ``script/code/clean.sh [source_code]`` with a for loop. So It is not automatic but manual. I can refrain to do this, but it will be better if we do the cleaning every time before pushing the code.

Reply:
I see, thanks for explaining!